### DISCUSSION TITLE
Local host 8080 doesnt start and gives error

### DISCUSSION THREAD

**Discussion by etripath** [apache/airflow#31026] [General]
### Apache Airflow version

2.6.0

### What happened

i tried to kill some previous sessions as my Airflow was always pointing to older config files. 

When i removed all airflow installation from my local laptop and reinstalled step by step , it doesnt open the http://localhost:8080/ anymore and gives the below error 

Ooops!
Something bad has happened.

Airflow is used by many users, and it is very likely that others had similar problems and you can easily find
a solution to your problem.

Consider following these steps:

  * gather the relevant information (detailed logs with errors, reproduction steps, details of your deployment)

  * find similar issues using:
     * [GitHub Discussions](https://github.com/apache/airflow/discussions)
     * [GitHub Issues](https://github.com/apache/airflow/issues)
     * [Stack Overflow](https://stackoverflow.com/questions/tagged/airflow)
     * the usual search engine you use on a daily basis

  * if you run Airflow on a Managed Service, consider opening an issue using the service support channels

  * if you tried and have difficulty with diagnosing and fixing the problem yourself, consider creating a [bug report](https://github.com/apache/airflow/issues/new/choose).
    Make sure however, to include all relevant details and results of your investigation so far.

Python version: redact
Airflow version: redact
Node: redact
-------------------------------------------------------------------------------
Error! Please contact server admin.

### How to reproduce

i am not sure 

### Operating System

Apple M1 Max

### Deployment

Other

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Don't create issues for basic troubleshooting please. Look at the logs to see the reason. There is nothing we can do with such report.

You missed that crucial step in the instructions:


> gather the relevant information (detailed logs with errors, reproduction steps, details of your deployment)

**Comment by potiuk** [comment]
Converted into discussion so that you can gather more information in case you would like to discuss it further.
